AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -412,7 +412,13 @@ - (void)setLogUrl:(NSString *)logUrl {
   @synchronized(self) {
     _logUrl = logUrl;
     if (self.channelGroup) {
-      [self.channelGroup setLogUrl:logUrl];
+      if (!self.appSecret) {
```

Comment:
Remove `!` and swap `if` and `else` block. It is unnecessary bit operation that you can remove.
```
if (!variable) {
  // Do A
} else {
  // Do B
}
```
can be rewritten to
```
if (variable) {
  // Do B
} else {
  // Do A
}
```